### ISSUE TITLE
Request: show number of violated rules when using auto-configuration

### ISSUE THREAD

**Issue by fubu** [jscs-dev/node-jscs#1125]
I'm a big fan of the new auto-configuration feature, but it would be even more helpful to me if its output would additionally show me the number of violated rules per preset as well. That'd give me an even clearer picture of which preset matches the tested codebase best. From my understanding of JSCS this information should be available when the auto-configuration finished testing against all presets?

**Comment by mrjoelkemp** [comment]
Hey @fubu! Thanks for the suggestion. Yes, that info is available. I originally considered adding that instead of `#Errors`, but there's no reason why both datapoints can't be shown.

Would you like to give it a go? We could use the help. You'd have to modify https://github.com/jscs-dev/node-jscs/blob/84ab5a3f2bcdc46c80eba5b17c00d5e01295be31/lib/config/generator.js#L105 to add the number of (unique) violated rules. Then modify https://github.com/jscs-dev/node-jscs/blob/84ab5a3f2bcdc46c80eba5b17c00d5e01295be31/lib/config/generator.js#L250 to add the column and https://github.com/jscs-dev/node-jscs/blob/84ab5a3f2bcdc46c80eba5b17c00d5e01295be31/lib/config/generator.js#L132 to add the data. Let me know if you have any questions; we're more than happy to help.

**Comment by fubu** [comment]
I just came up with another idea which would help me make better decisions when auto-configuring. Although the initial approach here supports me in choosing the right preset, the tool does not assist me when I have to decide whether to create an exception or to fix the issue myself: It just tells me which rules my codebase violates, but gives no clue how bad it really is. Maybe it only affects a few places or a single file (so it may be easy for me to fix or ignore), or maybe it's a more general problem (so I probably want to add an exception)?

In short: When asking for a decision per violated rule, tell the user the amount of errors and files flagged by this specific rule.

@mrjoelkemp I could add this on top of the existing pull request if that's okay with you guys.

**Comment by mrjoelkemp** [comment]
Great idea. +1
